pkg/sdkserver/sdkserver.go

```diff
@@ -144,7 +144,7 @@ func NewSDKServer(gameServerName, namespace string, kubeClient kubernetes.Interf
 		s1 := fields.OneTermEqualSelector("metadata.name", gameServerName)
 		opts.FieldSelector = s1.String()
 	}
-	factory := externalversions.NewSharedInformerFactoryWithOptions(agonesClient, 30*time.Second, externalversions.WithNamespace(namespace), externalversions.WithTweakListOptions(tweakListOptions))
+	factory := externalversions.NewSharedInformerFactoryWithOptions(agonesClient, 0, externalversions.WithNamespace(namespace), externalversions.WithTweakListOptions(tweakListOptions))
```

Comment:
Next step: wrap this in a feature gate method with the new feature gate, so that the when true, resync is 0, and when disabled (currently the default), it's still 30s.

https://github.com/googleforgames/agones/blob/25e0c4a3e85688d181b5f155d1ef66b576d839bc/pkg/util/runtime/features.go#L204

Is the function.

For example:

https://github.com/googleforgames/agones/blob/065052a9beead20bbc2d7bf9a18a7906bc73a882/pkg/sdkserver/sdk.go#L78-L87